test/SerialIOTest.cpp

```diff
@@ -26,6 +26,60 @@
 
 using namespace openPMD;
 
+namespace {
+    // only needed until we require C++14 and newer (201402L+)
+    template<typename T, typename... Args>
+    std::unique_ptr<T> my_make_unique(Args&&... args) {
+        return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
+    }
+}
+
```

Comment:
Not needed anymore :)
```suggestion
```

Reply:
Ah, that explains why there was a merge conflict..